README.md

```diff
@@ -0,0 +1,4 @@
+# rosidl
+
+```rosidl``` is one of the ros_core packages.
+See [documentation](http://docs.ros2.org/eloquent/developer_overview.html#the-rosidl-repository) for details of this package.
```

Comment:
I suggest pointing to the "latest" documentation so that we don't have to change this reference for each future release.

http://docs.ros2.org/latest/developer_overview.html#the-rosidl-repository